src/main/java/com/blackducksoftware/integration/jira/common/BlackDuckDataHelper.java

```diff
@@ -180,31 +179,20 @@ private int getSumOfRiskCounts(final List<RiskCountView> vulnerabilityCounts) {
         return null;
     }
 
-    public String getFirstLink(final HubView view, final String linkKey) throws IntegrationException {
-        return blackDuckService.getFirstLink(view, linkKey);
+    public Optional<String> getFirstLink(final BlackDuckView view, final String linkKey) {
+        return view.getFirstLink(linkKey);
```

Comment:
Now that the view itself contains the method to get the link, there is no need to wrap this.